### PR TITLE
✨(marsha) add new cronjob managing clean_aws_elemental_stack command

### PR DESCRIPTION
**Pull request by lunika**
## Purpose

There is a new management command clean_aws_elemental_stack that should be managed by a cronjob in our k8s.

## Proposal

- [x] Add new cronjob managing clean_aws_elemental_stack command

### CODE REVIEW

**Review by jmaupetit — COMMENTED** [review]
Given the growing number of almost identical cronjob objects you create in this app, what about creating a cronjob list instead and moving executed commands / scheduling in a configuration variable as it's achieved in Ralph's Tray:

- the template: https://github.com/openfun/ralph/blob/master/src/tray/templates/services/app/cronjob_pipeline.yml.j2
- the configuration: https://github.com/openfun/ralph/blob/master/src/tray/vars/all/main.yml#L16-L29

File: apps/marsha/vars/all/main.yml [hunk 1]
```diff
@@ -75,3 +65,32 @@ marsha_postgresql_resources:
   requests:
     cpu: 10m
     memory: 100Mi
+
+# -- Cronjobs
+marsha_deploy_clean_medialive_dev_stack_cronjob: false
```
Comment (jmaupetit) [inline apps/marsha/vars/all/main.yml]: Is this variable used elsewhere?

Comment (lunika) [reply apps/marsha/vars/all/main.yml]: In the project no, but this is the only redefine by environment

File: apps/marsha/vars/all/main.yml [hunk 2]
```diff
@@ -75,3 +65,32 @@ marsha_postgresql_resources:
...
+  - name: "clean-medialive-dev-stack"
+    schedule: "*/5 * * * *"
+    command: "clean_medialive_dev_stack"
+    resources: "{{ app_resources }}"
+    enabled: {{ marsha_deploy_clean_medialive_dev_stack_cronjob }}
+  - name: "clean-mediapackages"
+    schedule: "0 5 * * *"
+    command: "clean_mediapackages"
+    resources: "{{ app_resources }}"
+    enabled: true
```
Comment (jmaupetit) [inline apps/marsha/vars/all/main.yml]: What about defining those in the deployment project and define an empty list as the default `marsha_cronjobs` value?

Comment (lunika) [reply apps/marsha/vars/all/main.yml]: Because I don't want to copy past this everywhere. This is the limit of this refacto IMO

Comment (lunika) [reply apps/marsha/vars/all/main.yml]: And this list is needed to have a complete marsha environment working.

**Comment by lunika** [comment]
@jmaupetit I removed my last commit, I have to investigate why I can't override the cronjob list then in matsuo. 

I made a backup of the commit in an other branch and will wotk on it later. 
Can you finish the review please considering I will do your suggestion later ?
Thanks

**Review by jmaupetit — APPROVED** [review]
Maybe declare a new issue for the WIP improvement!

**Pull request merged**